hack/build-local-images.py

```diff
@@ -227,4 +236,4 @@ def debug(message):
     print "[ERROR] This script knows how to build:\n\t{}".format(
         "\n\t".join(map(full_name, image_config.keys()))
     )
-    exit(1)
\ No newline at end of file
+    exit(1)
```

Comment:
It was my editor running the auto-format hook that I run for the gocode (since that format is enforced).  I hadn't really intended to make the change, but when I looked at the diff it made it consistent, so I left it in.  Anyway, the new pull skips that change.  I may submit a separate commit in my next PR to clean the format.